src/Model/Event.php

```diff
@@ -412,10 +412,12 @@ public static function getItemArrayForImportedId(int $event_id, array $item = []
 	public static function getStrings(): array
 	{
 		// First day of the week (0 = Sunday).
-		$firstDay = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'first_day_of_week', 0);
+		$firstDay		= DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'first_day_of_week', 0);
+		$calendar_defaultView = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'calendar_defaultView', 'month');
```

Comment:
Please move `'system', 'calendar_defaultView'` to `'calendar', 'default_view'`.

Please also move `'system', 'first_day_of_week'` to `'calendar', 'first_day_of_week'`. For backward compatibility reasons, you'll have to keep referring to the previous config value.
```suggestion
		$firstDay    = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'calendar', 'first_day_of_week') ??
		               DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'first_day_of_week') ?? 0;
		$defaultView = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'calendar_defaultView', 'month');
```
But when you save the value, it should only set `'calendar', 'first_day_of_week'`.